docker-compose.yml

```diff
@@ -27,30 +27,26 @@ services:
       - "8000:8000"
     depends_on:
       - db
-    env_file: ./.env
```

Comment:
Updating the branch should remove these changes, I think dev already has these changes